File: domain/src/main/java/no/nav/melosys/domain/behandlingsgrunnlag/data/FysiskArbeidssted.java
package no.nav.melosys.domain.behandlingsgrunnlag.data;

import no.nav.melosys.domain.dokument.adresse.StrukturertAdresse;

public class FysiskArbeidssted {
    public String virksomhetNavn;
    public Boolean arbeidUtlandHjemmekontor;
    public StrukturertAdresse adresse = new StrukturertAdresse();
}

File: domain/src/main/kotlin/no/nav/melosys/domain/mottatteopplysninger/data/arbeidssteder/FysiskArbeidssted.kt
package no.nav.melosys.domain.mottatteopplysninger.data.arbeidssteder

import no.nav.melosys.domain.adresse.StrukturertAdresse


data class FysiskArbeidssted(
    var virksomhetNavn: String? = null,
    val adresse: StrukturertAdresse = StrukturertAdresse()
)


